packages/sdk/src/index.ts

```
@@ -295,11 +301,13 @@ export async function init<T extends SchemaType>(
         onTokenBalanceUpdated: (
             contract_addresses: string[],
             account_addresses: string[],
+            token_ids: string[],
             callback: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)